sns/cm/st/repair_cli.c

```diff
@@ -42,8 +42,8 @@ const char *srv_ep_addr[MAX_SERVERS];
 M0_INTERNAL int repair_client_init(void)
 {
 	int rc;
-
-	rc = m0_net_domain_init(&cl_ndom, &m0_net_lnet_xprt);
+	struct m0_net_xprt *xprt = m0_net_xprt_default_get();
+	rc = m0_net_domain_init(&cl_ndom, xprt);
```

Comment:
Used once. Will not further comment on this, please fix all such instances.

Reply:
incorporated